pkg/usagemetrics/googleanalytics.go

```diff
@@ -0,0 +1,79 @@
+package usagemetrics
+
+import (
+	"github.com/golang/glog"
+	analytics "github.com/jpillora/go-ogle-analytics"
+	k8sapi "github.com/openebs/maya/pkg/client/k8s/v1alpha1"
+	openebsversion "github.com/openebs/maya/pkg/version"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sversion "k8s.io/client-go/pkg/version"
+)
+
+const (
+	// GAclientID is the unique code of the project in Google Analytics
+	GAclientID = "UA-127388617-1"
+)
+
+// Event is a custom metric collected by OpenEBS developers
+type Event struct {
+	category string
```

Comment:
add 2 more parameters here called - type and id. e.type will be set as ApplicationName and e.id will be set up as PageTitle.

Reply:
Great, this will help us include hit types where hit type isn't `event`

Reply:
[Update] This might not be required, as currently we are only firing `Hit Types` of `event` and not `pageview` or something else. If required, I can create a `Hit` structure and a NewHit() which will initialize Event struct. More thinking might be required here.